src/libs/actions/BankAccounts.js

```diff
@@ -816,6 +816,27 @@ function updateReimbursementAccountDraft(bankAccountData) {
     Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT_DRAFT, bankAccountData);
 }
 
+/**
+ * Checks the given number is a valid US Routing Number
+ * @param {String} number
+ * @returns {Boolean}
+ */
+function validateRoutingNumber(number) {
```

Comment:
Hmm @parasharrajat I see your point, but I think this validation function is only needed here for now so we can keep it here. I think `validationUtils` is very useful for reusable validation functions. What do y'all think?

Reply:
I think validations should be kept in a separate file.
* validation functions are pure.
* They should be reusable and coded with reusability in mind.
* App could reuse the same function if we need to verify routing number again.
* It would be the same lines of code.
* It does not affect UI or business. Thus can be abstracted into lib level.

These were my thoughts behind that suggestion. But yeah looking at the use, it can stay here as well.